srcpkgs/sbs/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'sbs'
+pkgname=sbs
+version=1.0.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="pkg-config"
+makedepends="imlib2-devel libXinerama-devel"
+depends="imlib2-devel libXinerama-devel"
```

Comment:
unnecessary, should be auto-detected

Reply:
Compilation fails on `xbps-src pkg sbs` when I exclude those. Am I missing something?

Reply:
remove them only from depends, not makedepends